### ISSUE TITLE
Dialog always contains scrollbar (V14.2)

### ISSUE THREAD

**Issue by knoobie** [vaadin/vaadin-dialog-flow#185]
After upgrading our application I noticed that every dialog now has a scrollbar. 

This can be reproduced with a starter created on vaadin.com/start with the following:

```java
    Button button = new Button("Click me", e -> {
        Dialog dialog = new Dialog();
        dialog.add(new H3("My Dialog"));
        dialog.add(new TextField("My Label 1"));
        dialog.open();
      });
    add(button);
```

Resulting in:
![image](https://user-images.githubusercontent.com/3968629/82439061-53cff480-9a9a-11ea-96f7-af4f50eeb161.png)

It looks like an issue related to the new resize feature: `<div id="resizerContainer" class="resizer-container">` with its CSS:

```css
.resizer-container { 
   height: 100%; 
   width: 100%; 
   overflow: auto; 
}
```
Removing `height` or `overflow` fixes it for me.

Versions:
* OS: Ubuntu 
* Vaadin 14.2.0
* Java 8
* Chrome 81 & Firefox 77

**Comment by web-padawan** [comment]
@knoobie thanks for the issue. Transferred it to https://github.com/vaadin/vaadin-dialog-flow

We can leave it there although the actual fix should be in the web component.

**Comment by knoobie** [comment]
Issue can be reproduced with 14.2.2 as well

**Comment by knoobie** [comment]
@tomivirkki when is this published in Vaadin 14? Vaadin 14.3.4 still has the same problem after months :(

**Comment by tomivirkki** [comment]
@knoobie Looks like this was never backported to V14. Sorry about that. Created the. PR just now https://github.com/vaadin/vaadin-dialog-flow/pull/219
